File: contest/atcoder_beginer_contest_205/power_compare.py
"""
author : @akash kumar
"""

a,b,c=map(int,input().split())
if c%2!=0:
  if a<b:
    print("<")
  elif a>b:
    print(">")
  else:
    print("=")
else:
   if abs(a)<abs(b):
      print("<")
   elif abs(a)>abs(b):
      print(">")
   else:
    print("=")

File: contest/atcoder_beginer_contest_205/power_compare.cpp
/* author : @akash */

#include<bits/stdc++.h>
using namespace std;

#define ll long long int
#define pb push_back
#define mod 1000000007
#define ld long double


void solve()
{
   int a;
   int b;
   int c;
   cin>>a>>b>>c;
   if(pow(a,c)<pow(b,c))
   {
   	cout<<"<";
   }
   else if(pow(a,c)>pow(b,c))
   {
   	cout<<">";
   }
   else
   {
   	cout<<"=";
   }
}

int main()
{
	ios_base::sync_with_stdio(0);
	cin.tie(0);
	cout.tie(0);
    solve();
    cout<<"\n";	    
	return 0;
}
